src/Magnum/Math/Functions.h

```diff
@@ -410,6 +411,33 @@ template<std::size_t size, class T> inline Vector<size, T> ceil(const Vector<siz
     return out;
 }
 
+
+/**
+@brief Computes a Binomial Coefficient
+
+Given n elements computes the number of ways to choose k elements
+without replacement. The number of ways to do this is
+given by
```

Comment:
Stealing further from the same [matlab docs](http://matlab.izmiran.ru/help/techdoc/ref/nchoosek.html), I think this would be the most concise explanation:

```
Returns the number of combinations of @f$ n @f$ things taken @f$ k @f$ at a time.
```